lib/dasher.js

```diff
@@ -31,6 +33,23 @@ function DasherButton(button) {
   doLog(button.name + " added.")
 }
 
+function doCommand(command, name, callback) {
+  const execFile = require('child_process').execFile;
```

Comment:
`require`s should go at the top of the file. There is no need to do this every time a command is invoked, and it will just slow things down.